agent/rpc_from.go

```diff
@@ -84,6 +84,11 @@ func (a *orbAgent) handleGroupRPCFromCore(client mqtt.Client, message mqtt.Messa
 
 }
 
+func (a *orbAgent) handleAgentStop(payload fleet.AgentStopRPCPayload) {
+	a.logger.Info(payload.Reason)
```

Comment:
let's prefix this with a msg
`a.logger.Info(fmt.Sprintf("control plane requested we terminate, reason: %s", payload.Reason))`